### PR TITLE
els structure for document uploader [delivers #158768739]

### PR DESCRIPTION
**Pull request by tinyels**
## Description

rudimentary document uploader has a route so that we can start adding additional functionality. A good deal of this was copied from current office orders info component. I pulled out a bit of what was shared, but expect that somethings will change more once we are loading the document list, so didn't want to overthink refactor.

## Reviewer Notes

there are e2e tests that should show how it loads, but you can also hit http://officelocal:4000/moves/0cec3ee0-87eb-44a8-aea2-b20fce06bf61/documents when the e2e tests are running

## Code Review Verification Steps

* [x] All tests pass.

## References

* [Pivotal story](https://www.pivotaltracker.com/story/show/158768739) for this change

## Screenshots

image

### CODE REVIEW

File: src/scenes/Office/DocumentViewer.jsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+DocumentViewer.propTypes = {
+  loadMoveDependencies: PropTypes.func.isRequired,
+};
+
+const mapStateToProps = state => ({
+  swaggerError: state.swagger.hasErrored,
+  ordersSchema: get(state, 'swagger.spec.definitions.CreateUpdateOrders', {}),
+  orders: state.office.officeOrders || {},
+  move: get(state, 'office.officeMove', {}),
```
Comment (stangah) [inline src/scenes/Office/DocumentViewer.jsx]: If you need `get` here, I think you'll also need it for most of the other props here

File: src/scenes/Office/DocumentViewer.jsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+              Something went wrong contacting the server.
+            </Alert>
+          </div>
+        </div>
+      );
+    return (
+      <div>
+        <div className="usa-grid">
+          <div className="usa-width-two-thirds orders-page-column">
+            <div style={{ minWidth: '400px' }}>
```
Comment (stangah) [inline src/scenes/Office/DocumentViewer.jsx]: Any reason you need this style to be inline and not part of the CSS?

File: src/scenes/Office/DocumentViewer.jsx (hunk 1 above)
Comment (tinyels) [reply src/scenes/Office/DocumentViewer.jsx]: This was cut and paste from OrdersInfo.jsx and will probably go away once we are pulling real data. However, could use same `||{}` as the proceeding line. State will never not have an officeMove (unless we make a major change in `appReducer.js`), so get is not necessary here.

File: src/scenes/Office/DocumentViewer.jsx (hunk 2 above)
Comment (tinyels) [reply src/scenes/Office/DocumentViewer.jsx]: Cause this is also a stub that has now died. ;)

**Pull request merged**